views/home.ejs

```diff
@@ -1,5 +1,6 @@
 <link rel="stylesheet" href="/css/home.css">
 <body>
+<input type="hidden" id="currentUserId" value="<%= currentUser ? currentUser._id : 'not_logged_in' %>" />
```

Comment:
Das ist bisschen dirty 😅, kannst du nicht die currentUser._id direkt in der favorite() function holen? Die ändert sich ja nicht in der Session. Ich weiss ejs mit js ist bisschen umständlich, aber du kannst vielleicht sowas hier verwenden: ``let js_var = JSON.parse(`<%- JSON.stringify(ejs_var) %>`);``